pylint/testutils/utils.py

```diff
@@ -2,8 +2,20 @@
 # For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
 # Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
 
+import contextlib
 import os
-from typing import List
+import sys
+from typing import Iterator, List, TextIO
+
+
+@contextlib.contextmanager
+def _patch_streams(out: TextIO) -> Iterator:
```

Comment:
```suggestion
def _patch_streams(out: TextIO) -> Iterator[None]:
    """Patch and subsequently reset a text stream."""
```
Do we want to keep this private?

Reply:
I think so, beside here's probably a way to use patch like I did for sys.path, I'm just not used to it.

Reply:
Ha shit I push forced without pulling the change.